docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,61 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
+
+<Admonition type="note" title="Secure Cipher Suites">
+  Teleport's RDP client supports only secure algorithms
+  for making TLS connections, so we have to configure a Windows server
+  to support those cipher suites as well.
+  This step is only *neccessary* for Windows Server 2012 R2 as it does not support
+  secure algorithms by default.
+</Admonition>
+
+
+In this step we'll create a new certificate template that uses
+elliptic curve cryptography, and then configure our GPO
+to use the newly created template to issue certificates used for Remote Desktop connections.
+
+# Create a certificate template
+
+1. Begin by navigating to:
+```
+Start > Control Panel > Administrative Tools > Certificate Authority
+```
+to open the CA Microsoft Management Console (MMC) GUI.
+2. Open your CA computer and right-click on `Certificate Templates`, then select `Manage`.
+3. Find `Computer` template on the list, right-click on it, then select `Duplicate Template`.
+4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.
+In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.
+5. Go to `General` tab and change `Template display name` to `RemoteDesktopAccess`. Make sure `Template name` is also `RemoteDesktopAccess`.
+6. In the `Cryptography` tab change `Provider Category` to `Key Storage Provider`, then `Algorithm name` to `ECDH_P384`. Also, change `Request hash` to `SHA384`.
+7. Next, in the `Extensions` tab select `Application Polices` and click `Edit` button.
+8. Remove all entries from the list.
+9. Click `Add…` button, then `New…`. Set the value of `Name` to `Remote Desktop Authentication`. Change the object identifier to `1.3.6.1.4.1.311.54.1.2.` and click `OK`.
```

Comment:
> (...) Change the object identifier to `1.3.6.1.4.1.311.54.1.2.`

What does `1.3.6.1.4.1.311.54.1.2.` mean? Could you add its "human" name in parenthesis, so people understand better what they are doing?

Reply:
This number is object identifier (OID) for Remote Desktop Access enhanced key usage extension, but this point will be removed completely. It turns out that we actually don't need to set it to make RDP working. The suggestion was good anyway. The human readable name should've been added if this point had to remain.